repos/os/include/net/ipv4.h

```diff
@@ -66,8 +66,8 @@ class Net::Ipv4_packet
 		 ** IPv4 header fields **
 		 ************************/
 
-		unsigned         _version         : 4;
 		unsigned         _header_length   : 4;
+		unsigned         _version         : 4;
```

Comment:
Ok, the problem is that although it is one byte (version and IHL) and byte-order is correct here, within bitfields the bit-order is not correctly reproduced here. As an interims fix the re-order is a good solution. In general, I agree to chelmuth that we should get rid of bitfields here and use the register framework (which did not exist yet when those headers got written).
Apart from that, it would be nice when the header length return value would be expressed in bytes instead of 32bit words.